src/test/java/de/tum/in/www1/artemis/programmingexercise/ProgrammingExerciseIntegrationTestService.java

```diff
@@ -270,6 +270,34 @@ void testProgrammingExerciseIsReleased_forbidden() throws Exception {
         request.get("/api/programming-exercises/" + programmingExercise.getId() + "/test-case-state", HttpStatus.FORBIDDEN, Boolean.class);
     }
 
+    void testExportSubmissionsByParticipationIds_excludePracticeSubmissions() throws Exception {
+        var repository1 = gitService.getExistingCheckedOutRepositoryByLocalPath(localRepoFile.toPath(), null);
+        var repository2 = gitService.getExistingCheckedOutRepositoryByLocalPath(localRepoFile2.toPath(), null);
+        doReturn(repository1).when(gitService).getOrCheckoutRepository(eq(participation1.getVcsRepositoryUrl()), anyString(), anyBoolean());
+        doReturn(repository2).when(gitService).getOrCheckoutRepository(eq(participation2.getVcsRepositoryUrl()), anyString(), anyBoolean());
+
+        // Set one of the participations to practice mode
+        assertThat(participation1.isTestRun()).isFalse();
```

Comment:
I think this assertion is not really part of the test, is it? Maybe you could just make sure that the first participation is not a test run using `participation1.setTestRun(true);` explicitly.
```suggestion
        participation1.setTestRun(true);
```

Reply:
Yeah, just wanted to sanity check that `participation1` is not a `testRun`. I changed it to a set, makes a bit more sense. Thanks!